core/src/main/java/com/redhat/lightblue/client/request/data/DataDeleteRequest.java

```diff
@@ -31,6 +31,7 @@ public JsonNode getBodyJson() {
         if (queryExpression != null) {
             node.set("query", queryExpression.toJson());
         }
+        appendExecutionToJson(node);
```

Comment:
You can reimplement the getBodyJson if an array is going to be used.

Is there a case an array will be used? I think it is all objects.